src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -14,7 +14,11 @@ set BUILDTOOLS_PACKAGE_DIR=%~dp0
 set MICROBUILD_VERSION=0.2.0
 set PORTABLETARGETS_VERSION=0.1.1-dev
 set ROSLYNCOMPILERS_VERSION=2.0.0-rc
-set MSBUILD_CONTENT_JSON={"dependencies": { "MicroBuild.Core": "%MICROBUILD_VERSION%", "Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%"},"frameworks": {"netcoreapp1.0": {},"net46": {}}}
+if [%BUILDTOOLS_USE_CSPROJ%]==[] (
+  set MSBUILD_PROJECT_CONTENT={"dependencies": { "MicroBuild.Core": "%MICROBUILD_VERSION%", "Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%"},"frameworks": {"netcoreapp1.0": {},"net46": {}}}
+) ELSE (
+  set MSBUILD_PROJECT_CONTENT=^^^<Project Sdk="Microsoft.NET.Sdk"^^^>^^^<PropertyGroup^^^>^^^<TargetFrameworks^^^>netcoreapp1.0;net46^^^</TargetFrameworks^^^>^^^<DisableImplicitFrameworkReferences^^^>true^^^</DisableImplicitFrameworkReferences^^^>^^^</PropertyGroup^^^>^^^<ItemGroup^^^>^^^<PackageReference Include="MicroBuild.Core" Version="%MICROBUILD_VERSION%" /^^^>^^^<PackageReference Include="Microsoft.Portable.Targets" Version="%PORTABLETARGETS_VERSION%" /^^^>^^^<PackageReference Include="Microsoft.Net.Compilers" Version="%ROSLYNCOMPILERS_VERSION%" /^^^>^^^</ItemGroup^^^>^^^</Project^^^>
```

Comment:
I cleaned up the formatting of both versions so it is much more legible now. Making these into standalone files is doable, but it's separate from the change here. We could have done it before; these files are not actually parameterized, all of the versions are hard-coded in the previous lines, so they could have been hard-coded in a separate file.